bridge/bridgeapp/models.py

```diff
@@ -10,11 +10,13 @@ class Thread(models.Model):
     """Thread table, includes ManyToMany fields for Categories"""
     body = models.TextField(unique=True)
     date = models.DateTimeField(auto_now_add=True)
+    # creates many to many relationship in db with Category table
     categories = models.ManyToManyField(Category)
 
 
 class Response(models.Model):
     """Responses to Threads table, includes ForeignKey to map Many to One for Threads"""
     body = models.TextField(unique=True)
+    # Django provided cascade method deletes all response objects when associated thread is deleted
```

Comment:
I would say:
```
# deletion of a thread will delete all the responses related to it
```
Also, though they're Richard's comments, I would edit all the docstrigs for the models to the format:
```
"""Response model with body, thread, date fields"""
```
relace Response with Category and Thread for the docstrings therefor